src/BenchmarksApps/TodosApi/AppSettings.cs

```diff
@@ -33,22 +33,10 @@ internal static class AppSettingsExtensions
 {
     public static IServiceCollection ConfigureAppSettings(this IServiceCollection services, IConfigurationRoot configurationRoot, IHostEnvironment hostEnvironment)
     {
-        // Can't use the configuration binding source generator due to bug where it emits non-compiling code right now
-        // https://github.com/dotnet/runtime/issues/83600
-        var optionsBuilder = services.Configure<AppSettings>(configurationRoot.GetSection(nameof(AppSettings)))
-            .AddOptions<AppSettings>();
-
-        if (!hostEnvironment.IsBuild())
-        {
-            services.AddSingleton<IValidateOptions<AppSettings>, AppSettingsValidator>();
-            optionsBuilder.ValidateOnStart();
-        }
```

Comment:
@DamianEdwards - I assume we still want the `!IsBuild()` part here.

Reply:
Yes we do